src/commonMain/kotlin/baaahs/Pinky.kt

```diff
@@ -484,6 +489,23 @@ class Pinky(
         }
     }
 
+    private inner class MdnsBrainListenHandler : Network.MdnsListenHandler {
+        override fun resolved(service: Network.MdnsService) {
+            val brainId = service.hostname
+            val address = service.getAddress()
+            if (address != null) {
+                val version = service.getTXT("version")
+                val idfVersion = service.getTXT("idf_ver")
+                val msg = BrainHelloMessage(brainId, null, version, idfVersion)
```

Comment:
[Kotlin style comment] If you make BrainHelloMessage a data class, we can use named arguments here for readability

Reply:
I’m disinclined to fuck with `BrainHelloMessage` right now. Maybe later when I get around to adding a bunch of new brain messages.